DQM/TrackingMonitorSource/python/StandaloneTrackMonitor_cfi.py

```diff
@@ -1,29 +1,28 @@
-    vertexTag         = cms.untracked.InputTag("selectedPrimaryVertices"),
-#    vertexTag         = cms.untracked.InputTag("offlinePrimaryVertices"),
-    puTag             = cms.untracked.InputTag("addPileupInfo"),
-    clusterTag        = cms.untracked.InputTag("siStripClusters"),
-    AlgoName          = cms.untracked.string('GenTk'),
-#    trackInputTag     = cms.untracked.InputTag('generalTracks'),
-    trackInputTag     = cms.untracked.InputTag('selectedTracks'),
-    offlineBeamSpot   = cms.untracked.InputTag('offlineBeamSpot'),
-    pfCombinedSecondaryVertexV2BJetTags = cms.untracked.InputTag("pfCombinedInclusiveSecondaryVertexV2BJetTags","","RECO"),
-    trackQuality      = cms.untracked.string('highPurity'),
-    TCProducer        = cms.untracked.InputTag("initialStepTrackCandidates"),
-    MVAProducers      = cms.untracked.vstring("initialStepClassifier1", "initialStepClassifier2"),
-    TrackProducerForMVA = cms.untracked.InputTag("initialStepTracks"),
-    doPUCorrection    = cms.untracked.bool(False),
-    isMC              = cms.untracked.bool(False),
-    puScaleFactorFile = cms.untracked.string("PileupScaleFactor_282037_ZtoMM.root"),
-    trackScaleFactorFile = cms.untracked.string("PileupScaleFactor_282037_ZtoMM.root"),
-    haveAllHistograms = cms.untracked.bool(True),
-    verbose           = cms.untracked.bool(False),
-    trackEtaH         = cms.PSet(Xbins = cms.int32(60), Xmin = cms.double(-3.0),Xmax = cms.double(3.0)),
-    trackPtH          = cms.PSet(Xbins = cms.int32(100),Xmin = cms.double(0.0),Xmax = cms.double(100.0)),
-    trackMVAH          = cms.PSet(Xbins = cms.int32(100),Xmin = cms.double(-1.0),Xmax = cms.double(1.0))
+from DQM.TrackingMonitorSource.standaloneTrackMonitorDefault_cfi import standaloneTrackMonitorDefault
+standaloneTrackMonitor = standaloneTrackMonitorDefault.clone(
+    moduleName        = "StandaloneTrackMonitor",
+    folderName        = "highPurityTracks",
+    vertexTag         = "selectedPrimaryVertices", # "offlinePrimaryVertices",
+    puTag             = "addPileupInfo",
+    clusterTag        = "siStripClusters",
+    AlgoName          = 'GenTk', # 'generalTracks',
+    trackInputTag     = 'selectedTracks',
+    offlineBeamSpot   = 'offlineBeamSpot',
+    trackQuality      = 'highPurity',
+    TCProducer        = "initialStepTrackCandidates",
+    MVAProducers      = ["initialStepClassifier1", "initialStepClassifier2"],
+    TrackProducerForMVA = "initialStepTracks",
+    doPUCorrection    = False,
+    isMC              = False,
+    puScaleFactorFile = "PileupScaleFactor_282037_ZtoMM.root",
+    trackScaleFactorFile = "PileupScaleFactor_282037_ZtoMM.root",
+    haveAllHistograms = True,
+    verbose           = False,
+    trackEtaH         = dict(Xbins = 60,  Xmin = -3.0, Xmax = 3.0),
+    trackPtH          = dict(Xbins = 100, Xmin =  0.0 ,Xmax = 100.0),
+    trackPhiH         = dict(Xbins = 100, Xmin = 3.15, Xmax = 3.15),
```

Comment:
@dbruschi 
this will lead to the merging problems: the same Xmin=Xmax will mean the range is dynamic